### ISSUE TITLE
Line numbering support and language name

### ISSUE THREAD

**Issue by rawb1** [johannschopplich/kirby-highlighter#6]
Hey Johann, 

Thank you for this great plugin ! 

I was looking for an easy solution to add line count and language name to my code blocks and on the [highlight.php](https://github.com/scrivo/highlight.php) repo and I found an [issue](https://github.com/scrivo/highlight.php/issues/28) giving the following solution : 

```php
$highlightedCode = '...';
$lines = preg_split('/\R/', $highlightedCode);
$output = '<span class="code-line">' . implode("</span>\n<span class=\"code-line\">", $lines) . '</span>';
```
 
So I edited the [HighlightAdapter.php](https://github.com/johannschopplich/kirby-highlighter/blob/main/classes/KirbyExtended/HighlightAdapter.php) file

added 2 lines for the line numbering
```php
$lines = preg_split('/\R/', $highlightedCode->value);
$highlightedCode = '<span class="code-line">' . implode("</span>\n<span class=\"code-line\">", $lines) . '</span>';
```

edited 1 line for the language name
```php
$codeNode->appendXML('<code data-language="'. $language.'">' . $highlightedCode . '</code>');
```

Both modifications don't change the default look (I haven't run the tests) and with some CSS it allows to easily display line number and language name in code blocks. 

```css
pre.hljs {
  line-height: 1.5;
  padding: 0.5rem 1rem;
  border-radius: 0.5rem;
  counter-reset: line;
}

pre.hljs > code::before {
  content: attr(data-language);
  float: right;
  color: gray;
}

pre.hljs .code-line {
  counter-increment: line;
}

pre.hljs .code-line::before {
  content: counter(line);
  display: inline-block;
  margin-right: 1rem;
  color: gray;
  opacity: 0.5;
}
```

<img width="725" alt="Screenshot 2021-04-10 at 12 26 01" src="https://user-images.githubusercontent.com/25453942/114266709-fae1b580-99f7-11eb-8769-a6a73f1ef1bd.png">

I wanted to share the info, but I can create a pull request if you like.

Have a great day,
Robin

**Comment by johannschopplich** [comment]
Hi Robin,

looks like a great idea. Please open a pull request. I will review then. 🙂

**Comment by johannschopplich** [comment]
Thanks again for both pull requests! Included in the current v1.4.0 release.
